angular-portfolio/src/app/bio/bio.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-bio',
+  templateUrl: './bio.component.html',
+  styleUrls: ['./bio.component.scss']
+})
+export class BioComponent implements OnInit {
+
+  constructor() { }
```

Comment:
Remove empty constructor and empty ngOnInit method. Also, remove "implements OnInit" from the class.